saw-core/src/Verifier/SAW/SharedTerm.hs

```diff
@@ -2622,6 +2623,31 @@ scUnfoldConstantSet sc b names t0 = do
           _ -> scTermF sc =<< traverse go tf
   go t0
 
+scUnfoldOnceFixConstantSet :: SharedContext
```

Comment:
This function is the thing that makes `unfold_fix_once` work, and it would be helpful to include a high-level comment describing how it works. If I understand what it is doing correctly, every time it sees something like this:

```
Prelude.fix a f

...

f : (a : sort 1) -> a -> a
f = ... f ...
```

Then if `f` is in the set of constants, this function will rewrite `Prelude.fix a f` to `... f ...` without unfolding `f` any further. Is that correct? If so, worth stating here.